CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+18.0.2 (Jun 15, 2021)
+----------
+* Added careers link to readme. [#1274](https://github.com/Shopify/shopify_app/pull/12274)
```

Comment:
Just tried the url `https://github.com/Shopify/shopify_app/pull/12274` which gave a 404.